modules/ROOT/pages/anypoint-insights.adoc

```diff
@@ -0,0 +1,504 @@
+
+----
+entity_type = 'APP' deployment_type = 'CLOUDHUB2'
+---- 
+
+This table lists supported filters:
+
+[%header,cols="10a,80a,10a"]
+|===
+| Filter | Operators | Description
+
+|`deployment_type`
+|`=`, `IN`
+|Filter by deployment target, such as CloudHub 2.0 (*CLOUDHUB2*) or Runtime Fabric (*RTF*), for example:
+----
+deployment_type IN ['CLOUDHUB2', 'RTF']
+----
+
+|`entity_type`
+|`=`, `IN`
+|Filter by deployment type (*APP* or *API*), for example:
+----
+entity_type = 'APP'`
+----
+
+|`entity`
+|`=`, `IN`
+|Filter by specific app or API deployments, for example:
+----
+entity IN ['99aea888-77de-6e06-5c55-d1234de56789', '09f99999-888d-7b77-6da5-bebd43210fc1']
+----
+
+|===
+
+`IN` captures exact matches to any comma-separated element in the array.
+
+//  H3: FILTER TIME PERIOD
+[[filter-by-time]]
+=== Filter Data by Time Period
+
+Use the time period selector (<<dashboard-overview, item 6>> in the example) to display data points collected for a given time period (such as the last 15 minutes or the last 24 hours). Your setting applies across all data in *Anypoint Insights* overview, including the *Entities List*.
+
+If you don't see data on the page, check that the selected time range is correct.
+
+//
+//H2: CHANGE ENVIRONMENT
+[[change-environment]]
+== Change the Environment
+
+Charts in Anypoint Insights provide metrics on a specific environment (such as Production or Sandbox). 
+
+To change current or default environment:
+
+. Open Anypoint Monitoring. 
+. Set a new environment:
+
+* To select an environment without changing the default, use the environment selector in Anypoint Monitoring, for example:
++
+image::insights-environment-selector.png["Environment selector in Anypoint Monitoring"]
+* To select a default environment:
+
+.. Click your profile icon (see <<profile, item 2>> in the example). 
+.. Select the name of your profile to open Access Management *Settings*.
+.. In *Settings*, select an environment as the default environment to view for the selected business group in Anypoint Monitoring, for example: 
++
+image::insights-default-selector.png["Default selector in Access Management"]
+.. Return to Anypoint Monitoring to check for the new business group setting.
++
+Refresh the page, if necessary. 
+
+//
+//H2: CHANGE BUSINESS GROUP
+[[change-bg]]
+== Change the Business Group
+
+Charts in Anypoint Insights display metrics for deployments in a specific business group. If you belong to more than one business group, you can change the selected group and the business group that Anypoint Monitoring selects by default.
+
+//do not change the order of numbers in the graphic or calloutlist
+[[profile]]
+image::insights-business-group-change.png["Business group selection options"]
+
+[calloutlist]
+. Business group selector 
```

Comment:
```suggestion
. Business-group selector 
```

Reply:
I am now doubting my compound-adjective edits...

Reply:
well, too late -- I'm going with it 😆